tests/fetchhead/nonetwork.c

```diff
@@ -406,3 +406,64 @@ void test_fetchhead_nonetwork__create_when_refpecs_given(void)
 	git_remote_free(remote);
 	git_buf_free(&path);
 }
+
+static int nrefs_found;
+static bool count_refs_called;
+
+int count_refs(const char *ref_name, const char *remote_url, const git_oid *oid, unsigned int is_merge, void *payload)
+{
+	const char *pfix = (const char *) payload;
+
+	GIT_UNUSED(remote_url);
+	GIT_UNUSED(oid);
+	GIT_UNUSED(is_merge);
+
+	count_refs_called = true;
+
+
+	if (!git__prefixcmp(ref_name, pfix))
+		nrefs_found++;
+
+	return 0;
+}
+
+void test_fetchhead_nonetwork__create_with_multiple_refspecs(void)
+{
+	git_remote *remote;
+	git_buf path = GIT_BUF_INIT;
+
+	cl_set_cleanup(&cleanup_repository, "./test1");
+	cl_git_pass(git_repository_init(&g_repo, "./test1", 0));
+
+	cl_git_pass(git_buf_joinpath(&path, git_repository_path(g_repo), "FETCH_HEAD"));
```

Comment:
Would be a bit easier if you moved down that line to where it's being used

Reply:
Done.